Tests/MapboxNavigationTests/URLDataCacheTests.swift

```diff
@@ -0,0 +1,87 @@
+import XCTest
+import TestHelper
+@testable import MapboxNavigation
+
+class URLDataCacheTest: TestCase {
+    let url = ShieldImage.i280.baseURL
+    let cache: URLDataCache = URLDataCache()
+
+    override func setUp() {
+        super.setUp()
+        self.continueAfterFailure = false
+        cache.urlCache.diskCapacity = 0
+        cache.clearCache()
+    }
+    
+    private func exampleResponse(with storagePolicy: URLCache.StoragePolicy) -> CachedURLResponse {
+        let data = Fixture.JSONFromFileNamed(name: "sprite-info")
+        let response = URLResponse(url: url, mimeType: nil, expectedContentLength: data.count, textEncodingName: nil)
+        let cachedResponse = CachedURLResponse(response: response, data: data, storagePolicy: storagePolicy)
+        return cachedResponse
+    }
+    
+    private func urlRequest(with url: URL) -> URLRequest {
```

Comment:
This method is duplicated three times. We can create `URLRequest` extension and move it there to prevent duplication.

Reply:
Thanks for the suggestion, the extension of `URLRequest.init(_:)` is added and the functions are removed.